mix.exs

```diff
@@ -31,6 +31,7 @@ defmodule Vtc.MixProject do
           "Frames Formats": [Frames.FeetAndFrames, Frames.TimecodeStr],
           "Seconds Formats": [Seconds.PremiereTicks, Seconds.RuntimeStr],
           "Source Protocols": [Seconds, Frames],
+          "Ecto Types": [Vtc.Ecto.Postgres.PgRational, Vtc.Ecto.Postgres.PgRational.Migrations],
```

Comment:
learned something new, had no clue documentation was defined in the mix file!

Reply:
Yeah, writing this lib forced me to a deep-dive on how ExDoc generated documentation haha